scene/gui/tree.cpp

```diff
@@ -3391,8 +3393,10 @@ void Tree::_go_down() {
 			return;
 		}
 
-		select_single_item(next, get_root(), selected_col);
-		queue_redraw();
+		if (selected_item) {
```

Comment:
Same here, in single select mode, going down select the next node after the root if nothing is selected.
I suggest moving `int col = selected_col < 0 ? 0 : selected_col;` before the `if (select_mode == SELECT_MULTI) {` and using `col` as the third parameter of `select_single_item`.

Reply:
Same as above. (ie. PR https://github.com/godotengine/godot/pull/92959 already takes this approach).